rosario_angelica/country.js

```diff
@@ -42,23 +44,25 @@ async function searchCountry() {
   }
 }
 
-// Function to display the details of the selected country
+
 function displayCountryDetails(details) {
-  const detailsHTML = details.map(detail => `<p><strong>${detail.label}:</strong> ${detail.value}</p>`).join('');
+  const detailsHTML = details.map(detail => `<p><strong>${detail.label}:
+  </strong> ${detail.value}</p>`).join('');
```

Comment:
I noticed that you've bring down the continuity of the line to lessen the column, that's great! However, I would like to ask a minor changes, the line 50  is the html code continuation of the line 49, you might need to add one tab indentation space before it. For example:
1. this does not show readability: ❌
![image](https://github.com/PUP-BSIT/exercise-14-sixth_sense/assets/146615920/5c445f58-cb2f-4e24-903a-e5f428198794)

2. You might want to add 1 tab indentation before the code in line 50, to show readability✅
![image](https://github.com/PUP-BSIT/exercise-14-sixth_sense/assets/146615920/d6b64ef3-815a-4b43-8646-e0dfb4f6eea1)